src/Experimental/ShortestPaths/ShortestPaths.jl

```diff
@@ -0,0 +1,105 @@
+module ShortestPaths
+using SparseArrays: sparse
+using LightGraphs
+using LightGraphs:AbstractPathState, DijkstraState, BellmanFordState,
+      FloydWarshallState, DEsopoPapeState, JohnsonState, AbstractGraph, AbstractEdge
+#
+# using LightGraphs: dijkstra_shortest_paths, bellman_ford_shortest_paths, floyd_warshall_shortest_paths,
+#     desopo_pape_shortest_paths, a_star, johnson_shortest_paths
+import Base:convert, getproperty
```

Comment:
Do we still overwrite these two methods?

Reply:
nope. Great catch.